schema/crdb/3.0.3/up.sql

```diff
@@ -1,17 +1,2 @@
-BEGIN;
-
-SELECT CAST(
-    IF(
-        (
-            SELECT version = '3.0.2' and target_version = '3.0.3'
-            FROM omicron.public.db_metadata WHERE singleton = true
-        ),
-        'true',
-        'Invalid starting version for schema change'
-    ) AS BOOL
-);
-
 ALTER TABLE omicron.public.ip_pool
     DROP COLUMN IF EXISTS internal;
-
-COMMIT;
```

Comment:
This is _so_ much better for the migration author anyway. The version check was clearly something to automate.